### DISCUSSION TITLE
⏳ 🔗 Cloudflare Turnstile integrstion into Elementor Forms

### DISCUSSION THREAD

**Discussion by Grassmedia** [elementor/elementor#27293] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Inability to use Cloudflare's Turnstile (ReCaptcha replacement)

### Describe the solution you'd like.

Ability to integrate Cloudflare Turnstile

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicocart** [comment]
Good topic. That's what I need

**Comment by DavePodosyan** [comment]
This is very much needed

**Comment by liv2day** [comment]
This already exists in https://wordpress.org/plugins/simple-cloudflare-turnstile/ right..?

**Comment by reedock** [comment]
This request is a duplicate of :
https://github.com/orgs/elementor/discussions/25671

**Comment by DavePodosyan** [comment]
I built a custom integration to add Cloudflare Turnstile support to Elementor Forms, following the same pattern Elementor uses for reCAPTCHA. To use it, you just need to include it in your theme's function.php file. 

https://gist.github.com/DavePodosyan/b4e6f0a261ce5c7ed3b30b0734d56291

**Comment by MorrowShore** [comment]
any news on this?
